src/backend/access/external/pxffilters.c

```diff
@@ -451,6 +452,51 @@ opexpr_to_pxffilter(OpExpr *expr, PxfFilterDesc *filter)
 	return false;
 }
 
+static List*
+append_attr_from_var(Var* var, List* attrs)
+{
+	AttrNumber varattno = var->varattno;
+	/* system attr not supported */
+	if (varattno > InvalidAttrNumber)
```

Comment:
@kavinderd, yes, physical attributes should have positive value, some system attributes have -1, and we are not projecting them.